--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .gitignore
	modified:   IsaacLab (new commits)
	modified:   source/isaaclab_marl/isaaclab_marl/managers/init_pos_manager.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	wks_logs/marl_soccer/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git commit ---
c003a180946a197435c20956e208d30bd7c9e0f9


--- git branch ---
main


--- git diff ---
diff --git a/.gitignore b/.gitignore
index 2c4ac5f..e5c6a80 100644
--- a/.gitignore
+++ b/.gitignore
@@ -34,3 +34,5 @@ _isaac_sim*
 _repo
 _build
 .lastformat
+
+IsaacLab
\ No newline at end of file
diff --git a/IsaacLab b/IsaacLab
index 7b26eb4..b5fa0eb 160000
--- a/IsaacLab
+++ b/IsaacLab
@@ -1 +1 @@
-Subproject commit 7b26eb4169e576e58592637d2dccf8c0316f376a
+Subproject commit b5fa0eb031a2413c182eeb54fa3a9295e8fd867c
diff --git a/source/isaaclab_marl/isaaclab_marl/managers/init_pos_manager.py b/source/isaaclab_marl/isaaclab_marl/managers/init_pos_manager.py
index 211f72b..6861c09 100644
--- a/source/isaaclab_marl/isaaclab_marl/managers/init_pos_manager.py
+++ b/source/isaaclab_marl/isaaclab_marl/managers/init_pos_manager.py
@@ -87,7 +87,7 @@ class InitPosManager:
 
     def re_sample_init_pos(self, env_ids=None):
         if env_ids is None:
-            env_ids = ...
+            env_ids = slice(None)
         if self.random_sampling:
             self.blue_default_pos[env_ids, ...] = self.uniform_sample(
                 self.blue_default_pos[env_ids, ...],
